batcher/aligned-sdk/src/sdk.rs

```diff
@@ -35,6 +35,18 @@ use futures_util::{
     StreamExt, TryStreamExt,
 };
 
+//Public constants for convenience
+pub const HOLESKY_PUBLIC_RPC_URL: &str = "https://ethereum-holesky-rpc.publicnode.com";
+
+const AGGREGATOR_GAS_COST: u128 = 400_000;
+const ADDITIONAL_SUBMISSION_GAS_COST_PER_PROOF: u128 = 13_000;
+const CONSTANT_GAS_COST: u128 = ((AGGREGATOR_GAS_COST * DEFAULT_AGGREGATOR_FEE_MULTIPLIER)
+    / DEFAULT_AGGREGATOR_FEE_DIVIDER)
+    + BATCHER_SUBMISSION_BASE_GAS_COST;
+const DEFAULT_AGGREGATOR_FEE_MULTIPLIER: u128 = 3; // to set the feeForAggregator variable higher than what was calculated
+const DEFAULT_AGGREGATOR_FEE_DIVIDER: u128 = 2;
+const BATCHER_SUBMISSION_BASE_GAS_COST: u128 = 125_000;
```

Comment:
duplication of these constants make the code difficult to maintain. If here is a good place for them to be held until we centralize all constants into some file, then remove them from other places and make them be imported from here